Documentation/NewContentElements/Index.rst

```diff
@@ -0,0 +1,52 @@
+.. include:: ../Includes.txt
+
+
+
+.. _new-content-elements:
+
+New Content Elements
+------------
+
+It's possible to add drop zone for new content elements in a custom places. There a special ContentEditableWrapperService.
```

Comment:
It's possible to add drop zones for new content elements in a custom content elements. This is done by the class called ContentEditableWrapperService.